mlflow/server/js/src/experiment-tracking/components/RunView.js

```diff
@@ -260,6 +261,16 @@ export class RunViewImpl extends Component {
 
         {/* Metadata List */}
         <Descriptions className='metadata-list'>
+          {runName !== '' && (
```

Comment:
We could always show a run ID:

<img width="1129" alt="image" src="https://user-images.githubusercontent.com/17039389/169432157-c401ecf8-feb6-4a35-a3a9-36cbcf78fc5e.png">